server/routes/classesRoutes.js

```diff
@@ -8,7 +8,7 @@ const responseStatus = require('../config/responseStatusConfig');
 router.get('/', async (req, res, next) => {
   try {
     const classes = await db.getAll();
-    res.status(responseStatus.success).json(classes);
+    res.status(responseStatus.success).json({classes});
```

Comment:
Are there brackets now because the shape of the data coming back now is different?

Reply:
The shape of the data is basically the same; it's just named now, so instead of receiving an unnamed object, like {name: biology, id: 1, ...etc} it will be classes: {name: biology, id: 1, ...etc}. I just found it clearer to reference on the frontend because before I couldn't say res.data.classes to reference that object because it wasn't named.